py/makemoduledefs.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+
+# This pre-processor parses provided objects' c files for 
+# `MP_REGISTER_MODULE(module_name, obj_module, enabled_define)`
+# These are used to generate a header with the required entries for 
+# `mp_rom_map_elem_t mp_builtin_module_table[]` in py/objmodule.c
+
+from __future__ import print_function
+
+import re
+import os
+import argparse
+
+
+def find_c_file(vpath, obj_file):
```

Comment:
That can be a nitpick, by first impression I have in my head looking at this line is question "why args in that order?" ;-)

Reply:
I hadn't really thought about it. 
On one hand, `vpath` had a list of root paths, with the obj_file being appending on the end of one of them, so the two are joined together in that visual order.
On the other hand, `obj_file` is kind of the point of the function, with vpath being the list of paths to search in.